clang/include/clang/Basic/TokenKinds.def

```diff
@@ -980,6 +983,9 @@ ANNOTATION(header_unit)
 // Annotation for end of input in clang-repl.
 ANNOTATION(repl_input_end)
 
+// Annotation for #embed
+ANNOTATION(embed)
```

Comment:
no, none of the type/template annotations are pairs. It might just be a thing for pragmas

Reply:
Yeah, pragmas tend to come in pairs because they have multiple tokens comprising the annotation. In this case, there's only one thing to annotate so there's only one annotation token (I originally had a start/end but refactored it away). So I think this is handled.